Tests/BeeDatasetTests/BeePPCATests.swift

```diff
@@ -42,9 +42,8 @@ final class BeePPCATests: XCTestCase {
     let initialLatent = Vector5(flatTensor: ppca.encode(frames[0].patch(at: obbs[0]).mean(alongAxes: [2])).flattened())
     let latentId = v.store(initialLatent)
 
-    fg.store(PPCATrackingFactor(poseId, latentId, measurement: frames[1].mean(alongAxes: [2]), W: W, mu: Tensor28x62x1(mu.squeezingShape(at:[0]))))
+    fg.store(AppearanceTrackingFactor<Vector5, Tensor28x62x1>(poseId, latentId, measurement: frames[1].mean(alongAxes: [2]), appearanceModel: ppca.generateWithJacobian))
```

Comment:
@marcrasi Could you take a look at this test? I think it's a shape mismatch inside optimization, but I can't figure out which is that.

Error is like:
```
Test Case 'BeePPCATests.testPPCATracking' started at 2020-09-24 17:29:09.996
Precondition failed: file SwiftFusion/FixedShapeTensor.swift, line 149
Current stack trace:
0    libswiftCore.so                    0x00007f23e02dd0a0 swift_reportError + 50
1    libswiftCore.so                    0x00007f23e0348f40 _swift_stdlib_reportFatalErrorInFile + 115
2    libswiftCore.so                    0x00007f23dffb2fee <unavailable> + 1621998
```